includes/MediaBynderStreamWrapper.inc

```diff
@@ -42,9 +29,6 @@ class MediaBynderStreamWrapper extends MediaReadOnlyStreamWrapper{
      *   The path to the thumbnail.
      */
     public function getWebimagePath() {
-        $parts = $this->get_parameters();
-        $url = media_bynder_embed_url($parts['i'], 'web');
-        return $url;
+        return @media_bynder_embed_url($this->parameters['f']);
```

Comment:
The problem is: they aren't handled in Drupal's code and the warnings are already visible before they reach our module's code